pkg/testframework/framework.go

```diff
@@ -159,6 +175,20 @@ func IsResourceAvailable(errChan chan error, client *Clientset, resource string)
 	errChan <- err
 }
 
+func AreConsoleResourcesUnavailable(client *Clientset) error {
+	errChan := make(chan error)
+	go IsResourceUnavailable(errChan, client, "ConfigMap")
+	go IsResourceUnavailable(errChan, client, "Route")
+	go IsResourceUnavailable(errChan, client, "Service")
+	go IsResourceUnavailable(errChan, client, "Deployment")
+	checkErr := <-errChan
+
+	if checkErr != nil {
```

Comment:
Is the `if` check necessary?  `if` we check `nil`, but return `nil` otherwise, can we just return `checkErr`?